website/app.py

```diff
@@ -137,8 +124,6 @@ def init_app(settings_module='website.settings', set_backends=True, routes=True,
     app.config['SESSION_COOKIE_SECURE'] = settings.SESSION_COOKIE_SECURE
     app.config['SESSION_COOKIE_HTTPONLY'] = settings.SESSION_COOKIE_HTTPONLY
 
-    if set_backends:
-        do_set_backends(settings)
```

Comment:
Might be good to raise a DeprecationWarning if `set_backends` is set, but it's not completely necessary.